Makefile

```diff
@@ -13,4 +16,33 @@ install-go-tools:
 
 .PHONY: format-code
 format-code: install-go-tools
-	goimports -w .
\ No newline at end of file
+	goimports -w .
+
+#----------------------------------------------------------------------------------
+# Tests
+#----------------------------------------------------------------------------------
+
+GINKGO_VERSION ?= 2.5.0 # match our go.mod
+GINKGO_ENV ?= GOLANG_PROTOBUF_REGISTRATION_CONFLICT=ignore ACK_GINKGO_DEPRECATIONS=$(GINKGO_VERSION)
+GINKGO_FLAGS := -v -tags=purego -compilers=4 -fail-fast -randomize-suites -randomize-all -skip-package=./installutils/kubeinstall,./debugutils/test
+GINKGO_REPORT_FLAGS := --json-report=test-report.json --junit-report=junit.xml
+GINKGO_COVERAGE_FLAGS := --cover --covermode=count --coverprofile=coverage.cov
+
+# This is a way for a user executing `make test` to be able to provide flags which we do not include by default
```

Comment:
Is it plausible a user might want to override/opt into some of the default flags too?
ie should all of these have `?=`?

Reply:
Yeah probably safest to make them all optional. I figure these are the flags that we always want to run, since they are run in CI, so I can't think of a good situtation where you wouldn't want these assigned. However, I'm up for more opt-in functionality so I like this proposal and will update